examples/client.rs

```diff
@@ -0,0 +1,39 @@
+use imap_codec::imap_types::{
+    command::{Command, CommandBody},
+    core::Tag,
+};
+use imap_flow::{
+    client::{ClientFlow, ClientFlowEvent, ClientFlowOptions},
+    stream::AnyStream,
+};
+use tokio::net::TcpStream;
+
+#[tokio::main(flavor = "current_thread")]
+async fn main() {
+    let stream = TcpStream::connect("127.0.0.1:12345").await.unwrap();
+
+    let (mut client, greeting) =
+        ClientFlow::receive_greeting(AnyStream::new(stream), ClientFlowOptions::default())
+            .await
+            .unwrap();
+    println!("received greeting: {greeting:?}");
+
+    client.enqueue_command(Command {
```

Comment:
`enqueue_command` returns a handle. If you assign it to a val and print it, then it might be more clear to the reader what the purpose of the handle inside `CommandSent` is.

This could be important because the handle is not IMAP concept, so even a developer that knows IMAP might not immediately understand its meaning.